db/sql/migrations/runtime.sql

```diff
@@ -19,11 +19,16 @@
   PARAMETERS: None
 */
 
-UPDATE "peers" SET "state" = 1, "clock" = NULL WHERE "state" != 0;
-
 -- Overwrite unconfirmed tables with state from confirmed tables
-DELETE FROM mem_accounts2u_delegates;
-INSERT INTO mem_accounts2u_delegates ("accountId", "dependentId") SELECT "accountId", "dependentId" FROM mem_accounts2delegates;
+DROP TABLE mem_accounts2u_delegates;
+CREATE TABLE mem_accounts2u_delegates AS TABLE mem_accounts2delegates;
+ALTER TABLE mem_accounts2u_delegates ADD CONSTRAINT "mem_accounts2u_delegates_accountId_fkey" FOREIGN KEY ("accountId") REFERENCES mem_accounts(address) ON DELETE CASCADE;
+CREATE INDEX "mem_accounts2u_delegates_accountId" ON mem_accounts2u_delegates("accountId");
+
+DROP TABLE mem_accounts2u_multisignatures;
+CREATE TABLE mem_accounts2u_multisignatures AS TABLE mem_accounts2multisignatures;
+ALTER TABLE mem_accounts2u_multisignatures ADD CONSTRAINT "mem_accounts2u_multisignatures_accountId_fkey" FOREIGN KEY ("accountId") REFERENCES mem_accounts(address) ON DELETE CASCADE;
+CREATE INDEX IF NOT EXISTS "mem_accounts2u_multisignatures_accountId" ON mem_accounts2u_multisignatures("accountId");
+
+
```

Comment:
Should not we just disable the indexes on table for data import and then enable them back and reindex the table, instead of this approach?

Reply:
Same for constraints...

```sql
ALTER TABLE TableName NOCHECK CONSTRAINT ConstraintName;
```

and later 

```sql
ALTER TABLE TableName CHECK CONSTRAINT ConstraintName;
```